scripts/zap-scanner.sh

```diff
@@ -5,13 +5,17 @@ set -o pipefail
 
 TARGET=$1
 ENVIRONMENT=$2
+TARGET_ENV=$3
 
 TARGET_DIR="$(pwd)/tdrs-$TARGET"
 CONFIG_FILE="zap.conf"
 REPORT_NAME=owasp_report.html
 
 if [ "$ENVIRONMENT" = "nightly" ]; then
-    APP_URL="https://tdp-$TARGET-staging.app.cloud.gov/"
+    APP_URL="https://tdp-$TARGET-$TARGET_ENV.app.cloud.gov/"
+    if [ "$TARGET_ENV" = "prod" ]; then
+        APP_URL="https://api-tanfdata.acf.hhs.gov/"
```

Comment:
the frontend for prod has different URL, so the frontend report [here](https://api-tanfdata.acf.hhs.gov/admin/security/owaspzapscan/10/change/) isnt scanning the right URL.

Reply:
Should it be https://tanfdata.acf.hhs.gov?

Reply:
APP_URL should be tanfdata.acf.hhs.gov for prod frontend and api-tanfdata.acf.hhs.gov for prod backend. maybe this can be handled dynamically as well?